### ISSUE TITLE
python.executeStream is not a function within the iPython window

### ISSUE THREAD

**Issue by antonywu** [yhat/rodeo#208]
In the lower left hand corner, I noticed that it is running iPython, so I simply typed `1+1` and pressed `Enter`. I got the following error

```javascript
Uncaught Exception:
TypeError: python.executeStream is not a function
    at EventEmitter.<anonymous> (/Applications/Rodeo.app/Contents/Resources/app.asar/src/desktop/main.js:133:14)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
    at EventEmitter.<anonymous> (/Applications/Rodeo.app/Contents/Resources/atom.asar/browser/api/lib/web-contents.js:85:27)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
```

**Comment by antonywu** [comment]
I can also repro this under editor window. I typed `1+1` and pressed `Cmd+Enter` Same error shows up right away

Current running python version is `2.7.9`, and libraries version are
`ipython==3.1.0
ipython-genutils==0.1.0
jupyter==1.0.0
jupyter-client==4.1.1
jupyter-console==4.0.3
jupyter-core==4.0.6
matplotlib==1.4.3`

I am currently running `virtualenv` on Mac El Capitan

**Comment by glamp** [comment]
fixed in 1.3.0
